lib/accessory.js

```diff
@@ -54,15 +64,19 @@ class Accessory {
   }
 
   setOutlet (uuid, powered, done) {
-    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {
+	  var that = this;
+    this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {	
+		that.outlets.map(function(outlet){
```

Comment:
Seems like this addition is only to find the name of the outlet that's been updated. The call `this.outlet(device.outlets, uuid)` essentially filters out the current outlet targeted. You could then cache the result of this call and use it for the log line, i.e.

```javascript
setOutlet (uuid, powered, done) {
  this.remoteOutlet(uuid).update({desired_state: {powered}}, () => {
    const outlet = this.outlet(device.outlets, uuid);
    this.log(`${outlet.name} set to state: ${powered}`);
    done(undefined, outlet);
  });
}
```